lib/opensusebasetest.pm

```diff
@@ -979,6 +979,9 @@ sub wait_boot {
     # shutting down or booting up
     $self->{in_wait_boot} = 1;
 
+    # wait grub page for powerVM
+    $self->wait_grub(bootloader_time => $bootloader_time) if (is_pvm);
```

Comment:
This looks much better! The only concern I have is about jobs which do not have grub timeout disabled. In the past we have experienced issues when grub menu was not detected properly in 10 seconds. So it might affect some tests. I've triggered https://openqa.suse.de/t4376308 to prove that this doesn't affect encrypted scenarios.

Reply:
I checked your case can pass